charts/actions-runner-controller/templates/controller.metrics.serviceMonitor.yaml

```
@@ -9,6 +9,9 @@ spec:
   endpoints:
     - path: /metrics
       port: metrics-port
+      scheme: https
```

@jstewart612 Hey. Awesome @AlexDHoffer has figured out that we can add `bearerTokenFile` to the endpoint to make it work!

```suggestion
      bearerTokenFile: /var/run/secrets/kubernetes.io/serviceaccount/token
      scheme: https
```

Would you mind confirming it and wrap this change into a `if end` block to make it configurable? Once done I'll quickly merge this